src/memray/_memray_test_utils.pyx

```diff
@@ -269,3 +269,12 @@ def allocate_cpp_vector(size_t size):
     v.reserve(nelems)
     return v.size()
 
+
+@cython.profile(True)
+def fill_cpp_vector(size_t size):
+    cdef vector[int] v;
+    cdef size_t nelems = <size_t>(size / sizeof(int))
```

Comment:
Is there some reason for this `<size_t>` cast that I'm not seeing?

The tests pass even if it's removed, and it shouldn't be necessary because we're already dividing one `size_t` by another, which always returns a `size_t`.

Reply:
I am getting this:

```
src/memray/_memray_test_utils.pyx:276:30: Cannot assign type 'double' to 'size_t'
```

Reply:
Looks like that most likely should have been `//` instead of `/` so that we get integer division - but whatever, casting to double and then back to size_t isn't wrong, just a bit weird.